bool status
bool isdoubleget
float64 place
float64 enemy_vel
float64 enemy_dis

